app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersScreen.kt

```diff
@@ -309,6 +310,7 @@ private val successData = listOf(
 
 @Preview
 @Composable
+@ShowkaseComposable(skip = true)
```

Comment:
Question: Why don't we have group field in @Preview annotation here and below?

Reply:
how would you propose to group it here ? 🤔 I honestly do not see any valid group for this, let me know if you have some idea

Reply:
I think that it's `full screen` or something like `app screen`. I meant we can't see this screen in components at all.

Reply:
hey @nowakweronika  I looked what is going on , and from what I see the Showkase is added only to the component module and this is also the reason why it is genereting only stuff from within that module, that is why you are not seeing stuff from the app module, let me know if that answers your question and we can merge this guy to the develop

Reply:
Ok, thanks!